packages/server/__tests__/email/email.test.js

```diff
@@ -251,7 +251,7 @@ describe('Email sender', () => {
                 toAddress: 'staff.user@test.com',
                 subject: 'USDR Grants Tool Access Link',
                 text: 'Your link to access USDR\'s Grants tool is https://api.grants.usdigitalresponse.org/api/sessions?passcode=83c7c74a-6b38-4392-84fa-d1f3993f448d. It expires in 30 minutes',
-                tags: ['notification_type=passcode', 'user_role=staff', 'organization_id=0', 'team_id=0', 'service=test-dd-service', 'env=test-dd-env', 'version=test-dd-version'],
+                tags: ['notification_type=passcode', 'user_role=staff', 'organization_id=0', 'team_id=0', 'service=test-dd-service', 'env=test-dd-env', 'version=test-dd-version', 'dd_trace_id=undefined', 'dd_span_id=undefined'],
```

Comment:
Given the other suggestion regarding conditional inclusion, I think we can leave the tests unmodified, since it would be pretty difficult to instrument tests with dummy trace IDs and span IDs.
```suggestion
                tags: ['notification_type=passcode', 'user_role=staff', 'organization_id=0', 'team_id=0', 'service=test-dd-service', 'env=test-dd-env', 'version=test-dd-version'],
```